admin/taxonomy/class-taxonomy-social-fields.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Class WPSEO_Taxonomy_Social_Tab
+ *
+ * This class parses all the values for the social tab in the Yoast SEO settings metabox
+ */
+class WPSEO_Taxonomy_Social_Fields extends WPSEO_Taxonomy_Fields {
+
+	/**
+	 * Setting the class properties
+	 *
+	 * @param stdClass $term    The current taxonomy.
```

Comment:
From WordPress 4.4 and onward this will be `WP_Term` objects, we cannot typehint this because it breaks older WP versions but we can fix the documentation.